db/migrations/20200206182618_rename-timestamp.js

```diff
@@ -0,0 +1,20 @@
+
+exports.up = function(knex) {
+  return Promise.all([
+    knex.schema.table('playlists',
+      function(table) {
+        table.renameColumn('created_at', 'createdAt');
+        table.renameColumn('updated_at', 'updatedAt');
+      })
+  ])
+};
+
+exports.down = function(knex) {
+  return Promise.all([
+    knex.schema.table('playlists',
+      function(table) {
+        table.renameColumn('createdAt', 'created_at');
+        table.renameColumn('updatedAt', 'updated_at');
+      })
+  ])
```

Comment:
Realized it might be easier to change the created_at and updated_at here

Reply:
Very cool!

Reply:
✅Rename created_at and updated_at properties